microsoft-edge/web-platform/password-reveal.md

```diff
@@ -31,8 +31,8 @@ The `password` input type in Microsoft Edge includes a **password reveal** contr
 
 By default, the **password reveal** button inserts into the Shadow DOM of all HTML `input` elements with the `type` set to `"password"`.  Starting with Microsoft Edge Version 87, users or [enterprises](/deployedge/microsoft-edge-policies#passwordrevealenabled) may disable this feature globally.  You, web designers and developers, should expect most Microsoft Edge users to have the default experience.
 
-
-<!-- ====================================================================== -->
+
+<!-- ====================================================================== -->

```

Comment:
Hmm...when I edited the page via browser, some line endings were fixed automatically. Should I revert these changes?

Reply:
@mikehoffms - I'll scope out

Reply:
Let's keep the changed line endings, that the tool made.

I wasn't able to reproduce that automatic change when I tried the "." browser-based VS Code.  I haven't determined what the "line ending" change amounts to.  Comment dividers are supposed to be using normal line endings already.  I didn't see such an auto-fix in a different PR when I used the browser-based VS Code.